vignettes/osdc.Rmd

```diff
@@ -14,6 +14,41 @@ knitr::opts_chunk$set(
 )
 ```
 
+This package serves two overarching purposes:
+
+1.  To provide an open-source, code-based algorithm to classify type 1
+    and type 2 diabetes using Danish registers as data sources.
+2.  To inspire discussions within the Danish register-based research
+    space on the openness and ease of use on the existing tooling and
+    registers, and on the need for an official process for updating or
+    contributing to existing data sources.
+
+To read up on the overall design of this package as well as on the
+algorithm, check out the `vignette("design")`. For more explanation on
+the motivations, rationale, and needs for this algorithm and package,
+check out the `vignette("explanation")`.
+
+## Usage
+
+First, let's load up the package, as well as dplyr.
+
 ```{r setup}
+library(dplyr)
 library(osdc)
 ```
+
+The core of this package depends on the list of variables within
+different registers that are needed in order to actually classify the
+diabetes status of an individual. This can be found in the dataset:
+
+```{r}
+variable_description
```

Comment:
I don't have variable_description in my workspace, so I can't see this. But maybe that's fine at this stage?

Reply:
Yea, it depends on #37 and #48